src/python/tests/appengine/handlers/cron/oss_fuzz_setup_test.py

```diff
@@ -316,6 +317,26 @@ def test_execute(self):
         'MANAGED = True\n'
         'ISSUE_VIEW_RESTRICTIONS = none\n')
 
+    job = data_types.Job.query(
+        data_types.Job.name == 'libfuzzer_asan_i386_lib3').get()
+    self.assertIsNotNone(job)
+    self.assertEqual(job.platform, 'LIB3_LINUX')
+    self.assertItemsEqual(job.templates, ['asan', 'libfuzzer'])
+    self.assertEqual(
+        job.environment_string, 'RELEASE_BUILD_BUCKET_PATH = '
+        'gs://clusterfuzz-builds-i386/lib3/lib3-address-([0-9]+).zip\n'
+        'FUZZ_LOGS_BUCKET = lib3-logs.clusterfuzz-external.appspot.com\n'
+        'CORPUS_BUCKET = lib3-corpus.clusterfuzz-external.appspot.com\n'
```

Comment:
we are sharing the corpus with x86_64, that is fine ?

Reply:
Yep.

Reply:
I probably need to ensure pruning is off for i386 though

Reply:
Did this.